.github/workflows/report_quiz/report_quiz.sh

```diff
@@ -17,7 +17,7 @@ escp(){
 # to fetch programming questions
 get_quiz_questions_answers() {
     curl https://quizapi.io/api/v1/questions -G -d apiKey=$QUIZAPI_KEY -d limit=1 -d difficulty=easy 2>/dev/null | \
-    jq '.[] | {question: .question, A: .answers.answer_a, B: .answers.answer_b, C: .answers.answer_c, D: .answers.answer_d, E: .answers.answer_e}'
+    jq '.[] | {question: .question, A: .answers.answer_a, B: .answers.answer_b, C: .answers.answer_c, D: .answers.answer_d, E: .answers.answer_e, F: .answers.answer_f}'
```

Comment:
It's not the main subject of this PR to add more option but make sure to add it in your PR documentation.